### DISCUSSION TITLE
Kanji to Keyword option for SRS

### DISCUSSION THREAD

**Discussion by fabd** [fabd/kanji-koohii#272] [1. Feedback & Feature Requests]
Here's a first draft of what the updated Spaced Repetition settings could look like.

I thought it'd make sense to put the "kanji to keyword" option at the top of the form.

![image](https://user-images.githubusercontent.com/169391/197231428-f25b6160-5466-4006-9a85-7253c6949ea0.png)

**Comment by FarmerBob** [comment]
Thank you for adding this feature!

What do you think of hiding the dictionary word on the front of the card when kanji to keyword is chosen?

Often times the dictionary word is a dead giveaway to the keyword, so showing on the front of the card removes the challenge.
